api/cmd/server/main.go

```diff
@@ -122,9 +122,11 @@ func main() {
 
 	// Inject middleware
 	caching := http.CacheHandler{Log: logger}
+	security := http.SecurityHandler{Log: logger, Env: settings}
 	cors := http.CORSHandler{Log: logger, Env: settings}
 	logging := http.LoggingHandler{Log: logger}
-	router := caching.Middleware(cors.Middleware(logging.Middleware(r)))
+	//router := caching.Middleware(cors.Middleware(logging.Middleware(r)))
```

Comment:
Minor nit, but would you remove this comment reflecting the previous version? Git will track that for us.

Reply:
This has been removed